src/annotator/test/guest-test.coffee

```diff
@@ -67,10 +67,16 @@ describe 'Guest', ->
     }
     selections = null
     guestConfig = {pluginClasses: {}}
+    highlighter = {
+      highlightRange: sinon.stub()
+      removeHighlights: sinon.stub()
+    }
+    htmlAnchoring = {
+      anchor: sinon.stub()
+    }
 
```

Comment:
This ensures that the mocks for the highlighter and HTML anchoring modules are initialized for each test, for consistency with how we set up mocks in other tests.